### PR TITLE
Edit and proofread docs

### LINKED ISSUE #280: Final proofreading of docs
After we've written everything, we should proofread the whole thing.

### PR DESCRIPTION
**Pull request by davidtheclark**
Closes #280.

I tried to read all the documentation and make corrections and adjustments, hopefully improvements. There is a lot of documentation!

@samanpwbb for review.

### CODE REVIEW

File: src/typography.css [hunk 1]
```diff
@@ -20,12 +23,14 @@ textarea {
...
 /**
  * Classes for font size and line height, lists, block quotes, code blocks, and more.
+ *
  * @section Type basics
  * @memberof Typography
  */
 
 /**
- * Standard text scale. All text size classes include `-mm`, `-ml`, and `-mxl` variations to target screen sizes.
+ * Apply the standard text scale.
```
Comment (samanpwbb) [inline src/typography.css]: "Apply the standard text scale" sounds weird

File: src/typography.css [hunk 2]
```diff
@@ -616,7 +636,9 @@ textarea {
 /** @endgroup */
 
 /**
- * Conveniently style large blocks of semantic markup. `.prose` is designed to only be used when content is generated by a markdown processor and you don't control over each element's class list.
+ * Conveniently style large blocks of semantic markup.
+ * `.prose` is designed to only be used when content is generated by a markdown processor and you don't have control over each element's class list.
```
Comment (samanpwbb) [inline src/typography.css]: I think elsewhere you removed the "." from class name in description copy

**Review by samanpwbb — CHANGES_REQUESTED** [review]
A couple tiny pieces of feedbsck. Overall looks great and is much more consistent.

**Comment by davidtheclark** [comment]
Thanks @samanpwbb. I made changes addressing your feedback.

**Review by samanpwbb — APPROVED** [review]

**Pull request merged**